openpype/modules/deadline/plugins/publish/submit_max_deadline.py

```diff
@@ -74,7 +74,6 @@ def get_job_info(self):
         job_info.ChunkSize = instance.data.get("chunkSize", 1)
```

Comment:
```suggestion
        job_info.ChunkSize = instance.data.get("chunkSize", self.chunk_size)
```
it would take the OP setting of chunksize if I dont get it wrong.
The code below definitely works with the OP setting(and more precise).
```suggestion
        job_info.ChunkSize = self.chunk_size 
```